microbench/pixi.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
```

Comment:
Ooh. I did not know about Pixi yet. It looks nice and I see it integrates with uv.

Reply:
I have to update the README to explain how this can be used. I needed that since I tried this with a Debian for which I can't run `sudo apt install ...`. `pixi shell` is a simple way to get the non Python dependencies.